src/index.ts

```diff
@@ -0,0 +1,41 @@
+import {LoggingBindings} from '@loopback/extension-logging';
+import {ApplicationConfig, TestBackendApplication} from './application';
+
+export * from './application';
+
+export async function main(options: ApplicationConfig = {}) {
+  const app = new TestBackendApplication(options);
+  await app.boot();
+  await app.start();
+
+  const url = app.restServer.url;
+  console.log(`Server is running at ${url}`);
+
+  // configure Logs
+  app.configure(LoggingBindings.COMPONENT).to({
+    enableFluent: false,
+    enableHttpAccessLog: true,
```

Comment:
I didn't find access log with config in `.env.example`,
How can I configure the app to see the access log?

Reply:
Yes, I did not add configuration in the application to expose access logs. 
In other to achieve that, a global interceptor can be injected to handle the HTTP access logs. 

A custom-built interceptor can be used or we can simply make use of Winston's  _WINSTON_HTTP_ACCESS_LOGGER_ available in `@loopback/logging`. 

You can potentially also use [Morgan](https://www.npmjs.com/package/morgan)